flake.nix

```diff
@@ -109,6 +115,54 @@
         };
 
         installer-iso = inputs.self.nixosConfigurations.installer.config.system.build.isoImage;
+
+        # To build and run the docker container use the following command:
+        # nix build .#docker-image && docker load < result && docker run --cgroupns=host --privileged -t --rm wind-tunnel-runner:latest
```

Comment:
Nomad explicitly does not support running an agent in a docker container: https://developer.hashicorp.com/nomad/docs/deploy/production/requirements#running-nomad-in-docker

However it does seem to work for now by using the host's cgroup and running in privileged mode.

I assume this opens some security vulnerability compared to a standard docker container.

See https://developer.hashicorp.com/nomad/docs/deploy/production/requirements#linux-capabilities

Reply:
As discussed, we're considering it acceptable to communicate this risk along with recommending this container.